docs/src/explanations/introduction.md

```diff
@@ -3,25 +3,23 @@ layout: docs
 title:  "Introduction"
 section: "chisel3"
 ---
-This document is a tutorial introduction to _Chisel_ (Constructing
-Hardware In a Scala Embedded Language).  Chisel is a hardware
+
+# An Introduction to Chisel
+
+_Chisel_ (Constructing
+Hardware In a Scala Embedded Language) is a hardware
```

Comment:
I know you're mainly just cleaning up `wiki-deprecated`, but this page was the introduction to the old Chisel Tutorial, it doesn't really make sense in its current form anymore. I think we should rephrase it a bit, maybe renaming it `Motivation` or something? Or put this as the content for the [Explanations home page](https://www.chisel-lang.org/chisel3/explanations.html)?

Also, not related to these changes but quotes `>` look really bad on the website (https://www.chisel-lang.org/chisel3/docs/wiki-deprecated/introduction.html) we need a better way to format them.

Reply:
Yeah, I did edit it slightly to get rid of some of the tutorial "flavor", but since the vast majority of it is commented out, and on the website we already have a "landing" screen we can put the intro stuff to, I think it makes sense to rename this Motivation.

Reply:
I was thinking maybe this "landing page" when you click on the `Chisel3` link from the main website: https://www.chisel-lang.org/chisel3/

Reply:
you mean move this motivational content to that landing page?

Reply:
right, 100% agree.